webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+
+
+module.exports = {
+    entry: path.resolve(__dirname, 'client', 'src', 'index.js'),
+    output: {
+        path: path.resolve(__dirname, 'public', 'dist'),
+        filename: 'bundle.js'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(jsx|js)$/,
+                exclude: /node_modules/, 
+                use: 'babel-loader'
```

Comment:
I am not entirely clear on the nuances of this difference, but Josh indicated this was a 'better' way to differentiate the rules:use:  

use: [
          {
            loader: 'babel-loader',
            options: {
              presets: ['@babel/preset-env', '@babel/preset-react'],
            },
          },
        ],

Reply:
I have the same presets in my .babelrc file, I'm pretty sure they do the same thing?

Reply:
You're right. I missed that.